### PR TITLE
change path

### PR DESCRIPTION
**Pull request by ArthurNguyen**
Changing the paths of both deratives and raw

### CODE REVIEW

File: eeg_flow/utils/bids.py [hunk 1]
```diff
@@ -92,4 +92,4 @@ def get_xdf_folder(
     root = ensure_path(root, must_exist=True)
     check_type(participant, (str,), "participant")
     check_type(group, (str,), "group")
-    return root / f"sub-{participant}-{group}"
+    return root / f"sub-{participant}-{group}" / "eeg"
```
Comment (mscheltienne) [inline eeg_flow/utils/bids.py:95]: So now you have 2 roots, either passive or active? Correct?

Comment (ArthurNguyen) [reply eeg_flow/utils/bids.py:95]: yes! which would be set from step0 by the user

Comment (mscheltienne) [reply eeg_flow/utils/bids.py:95]: OK, that's good then. Why the extra "eeg" here?

Comment (ArthurNguyen) [reply eeg_flow/utils/bids.py:95]: we're starting to use the behavioral files as well (videos and olstat), so I was thinking of placing them in the raw subfolder of each participant too

>sub-P01-G1
>>eeg
>>beh

**Comment by ArthurNguyen** [comment]
thanks 🙏

**Pull request merged**